.github/workflows/django.yml

```diff
@@ -23,6 +35,11 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -r requirements.txt
+    - uses: harmon758/postgresql-action@v1
```

Comment:
isn't this step redundant with the postgres service you set up on line 17? In https://github.com/Harmon758/postgresql-action it says its sets up a PostgreSQL and that's what you do up there. Also, specifying different versions.

Reply:
It seems so by checking the build's output. Unfolding the `Initialize containers` step, you can see how it pulls and boots the PostgreSQL image with the last log line being "postgres service is healthy." Then, the step `Run harmon758/postgresql-action@v1` pulls postgres 11 but it fails to start due to "Bind for 0.0.0.0:5432 failed: port is already allocated."